.github/workflows/python.yml

```diff
@@ -33,7 +39,37 @@ jobs:
           poetry run pytest --cov=. --cov-report xml tests
       - name: SonarCloud Scan
         uses: SonarSource/sonarcloud-github-action@master
-        if: ${{ matrix.python-version == '3.11' }}
+        if: ${{ matrix.python-version == '3.11' && env.SONAR_TOKEN != '' }}
```

Comment:
Any reason for this change?

Reply:
So it's possible to skip this step easily when testing the release workflow in a fork - without having to set up Sonar Cloud for the fork.